crates/transaction-emitter-lib/src/emitter/account_minter.rs

```diff
@@ -605,4 +603,4 @@ pub async fn execute_and_wait_transactions(
     Ok(())
 }
 
-const CREATION_PARALLELISM: usize = 200;
+const CREATION_PARALLELISM: usize = 100;
```

Comment:
It would be great to make these configurable per instance instead of hardcoded

Reply:
creation is generally the same for all instances, so I don't think it needs to be configurable.

it was flaky across many tests, but rarely.